extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -28,7 +28,13 @@ public function testRenderInfobox( $input, $expectedOutput, $description ) {
 		$actualDOM->loadXML($actualOutput);
 		$expectedDOM->loadXML($expectedOutput);
 
-		$this->assertEquals( $expectedDOM->saveXML(), $actualDOM->saveXML(), $description );
+		// lets sanitize expected and actual HTML to not deal with whitespaces
```

Comment:
without sanitizing, I had something like this:
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 '<?xml version="1.0"?>
 <aside class="portable-infobox">
-  <footer class="portable-infobox-footer portable-infobox-item-margins portable-infobox-header-background portable-infobox-header-font">Footer value</footer>
+  <footer class="portable-infobox-footer portable-infobox-item-margins portable-infobox-header-background portable-infobox-header-font">
+	Footer value
+</footer>
 </aside>
 '